src/core/obj.js

```diff
@@ -712,41 +735,43 @@ var Catalog = (function CatalogClosure() {
             }
           }
           // The 'NewWindow' property, equal to `LinkTarget.BLANK`.
-          var newWindow = action.get('NewWindow');
+          const newWindow = action.get('NewWindow');
           if (isBool(newWindow)) {
             resultObj.newWindow = newWindow;
           }
           break;
 
         case 'Named':
-          var namedAction = action.get('N');
+          const namedAction = action.get('N');
           if (isName(namedAction)) {
             resultObj.action = namedAction.name;
           }
           break;
 
         case 'JavaScript':
-          var jsAction = action.get('JS'), js;
+          const jsAction = action.get('JS');
+          let js;
+
           if (isStream(jsAction)) {
             js = bytesToString(jsAction.getBytes());
           } else if (isString(jsAction)) {
             js = jsAction;
           }
 
           if (js) {
-            // Attempt to recover valid URLs from 'JS' entries with certain
-            // white-listed formats, e.g.
+            // Attempt to recover valid URLs from `JS` entries with certain
+            // white-listed formats, e.g.:
```

Comment:
Nit: It might be a personal preference, but I find that a colon after a period looks really ugly; can we please not do that :-)